tests/frontend/test_io.py

```diff
@@ -0,0 +1,543 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""Unit tests for the IO module"""
+import pytest
+
+pytestmark = pytest.mark.frontend
```

Comment:
the linter doesn't complain about this?

Reply:
It does ('all imports should come at the top of the file'), but this has always looked cleaner to me. Want me to move it down to below the imports?

Reply:
yup

Reply:
done